addons/l10n_ec/models/account_move.py

```diff
@@ -134,44 +133,29 @@ class AccountMove(models.Model):
         string="Payment Method (SRI)",
     )
 
-    def _get_l10n_ec_identification_type(self):
```

Comment:
You can't change the existing methods in stable version because it breaks the stable policy.

Reply:
A few lines below is the instruction
`_get_l10n_ec_identification_type = _get_l10n_ec_ats_identification_type`
Therefore the signature hasn't changed. The definition itself has, because it was not correct.